src/web/app/pages-session/session-submission-page/session-submission-page.component.html

```diff
@@ -77,7 +77,7 @@ <h4>Previewing Session as
 <tm-question-submission-form *ngFor="let questionSubmissionForm of questionSubmissionForms; let i = index; trackBy: trackQuestionSubmissionFormByFn"
                              [attr.id]="questionSubmissionForm.feedbackQuestionId"
                              [(formModel)]="questionSubmissionForms[i]" [formMode]="getQuestionSubmissionFormMode(questionSubmissionForm)"
-                             [isDisabled]="isSubmissionFormsDisabled"
+                             [isDisabled]="isSubmissionFormsDisabled && !this.previewAsPerson"
```

Comment:
Shouldn't we just remove this line?

https://github.com/TEAMMATES/teammates/blob/5deb81a13108950736b7484f938f1f989f962ce2/src/web/app/pages-session/session-submission-page/session-submission-page.component.ts#L147-L150

Reply:
Removing that will cause the "Submit Feedback" button to be enabled

Reply:
Then I think we should update the `[disabled]` attribute in the submit button to be `xx || this.previewAsPerson`. The current looks weird as `isSubmissionFormsDisabled` is controlled by `previewAsPerson` while `isDisabled` also depends on `previewAsPerson`.

Reply:
@xpdavid the submit button also depends on `isSubmissionFormsDisabled` so in this case no alternative is better than the other